packages/cli/src/lib/bundler/config.ts

```diff
@@ -124,17 +137,94 @@ export function createConfig(
         ? '[name].chunk.js'
         : '[name].[chunkhash:8].chunk.js',
     },
-    optimization: optimization(options),
     plugins,
+  };
+}
+
+export function createBackendConfig(
+  paths: BundlingPaths,
+  options: BackendBundlingOptions,
+): webpack.Configuration {
+  const { checksEnabled, isDev } = options;
+
+  const { loaders } = transforms(options);
+
+  return {
+    mode: isDev ? 'development' : 'production',
+    profile: false,
+    ...(isDev
```

Comment:
Tbh I think we could write this config with the assumption that we're in dev. I don't think there's any use-case for production here?

Reply:
Yep, will remove it